src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -126,7 +129,7 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 		for (const mimeType of Object.keys(output.data)) {
 			// Don't use a renderer for non-widget MIME types
 			if (mimeType === 'text/plain' ||
-				mimeType === 'text/html' ||
+				// mimeType === 'text/html' ||
```

Comment:
Is this intentional? I checked out this branch and it still works. I also tried a bunch of other Python and R interactive plots and viewer widgets and they all work too.

Since there is a builtin renderer for `text/html` it means that the call to `createRawHtmlOutput` below will never be reached. It also means that the webview produced by this function should always have the "webviewPreloads" script injected so we don't need the similar handlers in `downloadUtils`.

We could also consider removing the for loop and call to `createRawHtmlOutput` and deprecating that method.

It works as expected though, so we can take this PR as an improvement and deal with the above later.

Reply:
It was not intentional! I added a new issue for removing `createRawHtmlOutput` which should be pretty straightforward #4407